github-wide.css

```diff
@@ -120,3 +120,8 @@ button.discussion-sidebar-toggle {
 .vcard-avatar {
   width: 230px !important;
 }
+
+/* Commits: extended message under "..." */
+.commit-desc pre {
+  max-width: inherit;
```

Comment:
Sounds reasonable to me. Try a few out and see how it looks across various window sizes and what not.